src/aead/chacha20_poly1305.rs

```diff
@@ -50,14 +50,12 @@ fn chacha20_poly1305_open(ctx: &[u64; aead::KEY_CTX_BUF_ELEMS],
          ad)
 }
 
-fn chacha20_poly1305_update(state: &mut [u8; POLY1305_STATE_LEN],
-                            ad: &[u8], ciphertext: &[u8]) {
+fn chacha20_poly1305_update(state: &mut [u8; POLY1305_STATE_LEN], ad: &[u8], ciphertext: &[u8]) {
     fn update_padded_16(state: &mut [u8; POLY1305_STATE_LEN], data: &[u8]) {
         poly1305_update(state, data);
         if data.len() % 16 != 0 {
             static PADDING: [u8; 16] = [0u8; 16];
-            poly1305_update(state,
-                            &PADDING[..PADDING.len() - (data.len() % 16)])
+            poly1305_update(state, &PADDING[..PADDING.len() - (data.len() % 16)])
```

Comment:
I guess this is due to rustfmt? We should remove these non-test changes and/or move them to another PR.